api/tests/routes/backoffice/finance_test.py

```diff
@@ -1105,7 +1119,8 @@ def test_create_comercial_gesture_incident_from_one_booking_without_deposit_bala
         assert action_history.authorUser == legit_user
         assert action_history.comment == "Origine de la demande"
 
-    def test_create_comercial_gesture_incident_from_one_booking_with_deposit_balance(
+    @pytest.mark.skip(reason="Temporarly hide commercial gesture creation from the backoffice")
```

Comment:
```suggestion
    @pytest.mark.skip(reason="Temporarily hide commercial gesture creation from the backoffice")
```
Fallait le faire partout haha

Reply:
Il y en a deux autres encore